lib/components/term.js

```diff
@@ -184,13 +184,35 @@ export default class Term extends Component {
   }
 
   getStylesheet(css) {
-    const blob = new Blob([`
-      .cursor-node[focus="false"] {
-        border-width: 1px !important;
-      }
-      ${css}
-    `], {type: 'text/css'});
-    return URL.createObjectURL(blob);
+    if(process.platform === 'win32') {
+      const scroll = new Blob([`
+        .cursor-node[focus="false"] {
+          border-width: 1px !important;
+        }
+        ::-webkit-scrollbar {
```

Comment:
I have a suggested simplifcation.

```js
getStylesheet(css) {
  const osSpecifcCss = process.platform === 'win32' 
    ? `/* win32 css things */`
    : ''
  ;

  return URL.createObjectURL(new Blob([`
    .cursor-node[focus="false"] {
       border-width: 1px !important;
    }
    ${osSpecificCss}
    ${css}
  `], {type: 'text/css'}));
} 
```

It's less repetitive and it makes it easy to slot in the os-specific bits.